### PR TITLE
dbgate: 5.2.8 -> 5.3.0

### PR DESCRIPTION
**Pull request by luftmensch-luftmensch**
## Description of changes
[Release notes](https://github.com/dbgate/dbgate/releases/tag/v5.3.0)

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
Please rebase

**Comment by luftmensch-luftmensch** [comment]
> Please rebase

What do you mean? There is only one commit. I don't have write access to fix conflicting files

**Comment by SuperSandro2000** [comment]
> I don't have write access to fix conflicting files

Yeah, I cannot merge it with this and resolving the merge conflicts via the web ui will create an ugly merge commit which we try to avoid.

In your local clone, add the nixos/nixpkgs repo as a remote (eg named upstream) and then rebase your branch on the upstream/master branch. While doing that, git will stop and prompt you to fix the merge conflict. After that is done, just git add the file and git rebase --continue. To not accidentally ping many people if something goes wrong, draft the PR before.

**Comment by luftmensch-luftmensch** [comment]
> > I don't have write access to fix conflicting files
> 
> Yeah, I cannot merge it with this and resolving the merge conflicts via the web ui will create an ugly merge commit which we try to avoid.
> 
> In your local clone, add the nixos/nixpkgs repo as a remote (eg named upstream) and then rebase your branch on the upstream/master branch. While doing that, git will stop and prompt you to fix the merge conflict. After that is done, just git add the file and git rebase --continue. To not accidentally ping many people if something goes wrong, draft the PR before.

 @SuperSandro2000  Should be resolved now! Let me know if there is other changes you are willing to add

File: pkgs/by-name/db/dbgate/package.nix
Comment (SuperSandro2000) [inline pkgs/by-name/db/dbgate/package.nix:29]:
```suggestion
    description = "Database manager for MySQL, PostgreSQL, SQL Server, MongoDB, SQLite and others";
```
we recently merged a treewide PR to fix those 😅

Comment (luftmensch-luftmensch) [reply pkgs/by-name/db/dbgate/package.nix]: Didn't know that. Fixed!

**Pull request merged**
